### ISSUE TITLE
Questions about isic2018

### ISSUE THREAD

**Issue by ZBWKHH** [mindflow-institue/DAEFormer#17]
Dear author,：
I have read your article on DAEFormer and tested the Synapse dataset using the weights you provided. The final data is consistent with what you proposed in the paper. I believe this is a great paper.
But when I used ISIC2018, the DSC ultimately only reached 85.7 and I referred to the link you provided：
https://github.com/xmindflow/deformableLKA/tree/main/2D/skin_code

I haven't made any modifications to some key codes:

Optimizer=optim SGD (Net. parameters(), lr=args. base_lr, momentum=0.9, weight_decay=0.0001)

Scheduler=optim. lr_scheduler ReduceLROnPlateau (optimizer,'min ', factor=0.5, patient=10)

Base_lr: 0.01 batch_size: 4 max_epoch: 300

Thank you very much
